src/main/java/backend/resource/TurboLabel.java

```diff
@@ -129,13 +219,13 @@ public boolean hasGroup() {
     }
 
     public String getName() {
-        if (getDelimiter(actualName).isPresent()) {
-            String delimiter = getDelimiter(actualName).get();
+        if (getDelimiter(fullName).isPresent()) {
```

Comment:
Do we still need this method now that we have getShortName? Same for getGroup. We can just return the group we store.